### ISSUE TITLE
🐞 [BUG] - CHANGING THE UI OF WEBSITE IN LIGHT MODE 

### ISSUE THREAD

**Issue by Dishika18** [jfmartinz/ResourceHub#605]
### Issue Verification

- [X] I have verified that this is a new bug report.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Describe the bug

As of now when we view the website in light mode  the Ul is not good and colour of the font is bit faded with is not properly visible in light mode.
PICS OF BUGS:- 
![Screenshot 2024-05-30 213144](https://github.com/jfmartinz/ResourceHub/assets/152963337/347b3f6a-cbf0-4d15-a4f9-08d8a1e1349f)

![Screenshot 2024-05-30 213159](https://github.com/jfmartinz/ResourceHub/assets/152963337/a0fd57a9-0641-4c69-bd57-d5a213506a3d)

I can fix this bug and make an inpressive UI. Please assign this issue to me under GSSoC'24.

### Expected behavior

I'll make the fonts look visible and impressive compared to now.

**Comment by jfmartinz** [comment]
I assigned the issue about the text in the footer not too visible. maybe ou can work on other part of the wbesite

**Comment by Dishika18** [comment]
Yeah ok, I will work on the rest of the part except footer if that's ok?

**Comment by Dishika18** [comment]
kindly add labels to it
